gcalcli/gcalcli.py

```diff
@@ -943,8 +947,7 @@ def _edit_event(self, event):
                 return
 
             elif val.lower() == 'c':
-                self.printer.msg('Color: ', 'magenta')
-                val = input()
+                val = get_input(self.printer, 'Color: ', VALID_COLOURS)
```

Comment:
I think we said we'd try to stick with U-less spelling in the code itself, while hoping to eventually support U-full spelling in U-full locales in the output.

Reply:
My approach (more from habit!) has been to use British spellings in the code but drop the UI in outward-facing UI. But happy to revert to drop the U in `VALID_COLOURS`, etc.

Reply:
I don't mind the international spelling, to be honest, but since the existing codebase has u-less references to color I'd like to be consistent with the precedent.